src/services/GhostService.ts

```diff
@@ -16,32 +16,37 @@ const formatPostsDate = (posts: PostOrPage[]): PostOrPage[] => {
   }
 
   return posts?.map((post: PostOrPage) => {
-    post.published_at = new Intl.DateTimeFormat('en-US', options)
+    post.published_at = new Intl.DateTimeFormat(lang, options)
       .format(new Date(post.published_at))
 
     return post
   })
 }
 
 class GhostService {
-  public static async getPostBySlug (slug: string): Promise<GhostPostResponse> {
-    const url = `${CONTENT_API_BASE_URL}posts/slug/${slug}/?key=${process.env.GHOST_CONTENT_API_KEY}`
-    return await ApiClient.get(url)
+  public static async getPostBySlug (slug: string): Promise<PostOrPage> {
+    const includeQuery = '?include=tags,read_time,authors'
+    const url = `${CONTENT_API_BASE_URL}posts/slug/${slug}/${includeQuery}&key=${process.env.GHOST_CONTENT_API_KEY}`
+
+    const { posts } = await ApiClient.get(url)
+
+    return posts?.length ? posts[0] : null
```

Comment:
`head(posts)` (de lodash)

Reply:
siempre tendrá .length, no?

Reply:
@NievesBorrero @susomejias Siempre intentaría evitar el uso de librerias externas si no hacen falta. (Si lodash ya está instalada, pues bueno...)

Con respecto a lo que dice Nieves, si, siempre tiene la propiedad `.length`, pero si la usas como lo hace Suso, internamente JS hace la comprobación con `>0, null, undefined, false`. Se puede ver en el siguiente pantallazo
![Screenshot 2021-02-15 at 10 52 37](https://user-images.githubusercontent.com/19331093/107930925-ecd56280-6f7b-11eb-93be-eb346286760a.png)

Reply:
ostras! genial pues!

Reply:
Respecto a lo de lodash, como queráis, pero es una librería que yo siempre tendría si vas a trabajar bastante con arrays y colecciones, en ese caso merece la pena, si no... podemos no incluirla

Reply:
Sip, la tengo incluida porque me facilitaba un montón algunas cositas muy específicas con arrays, aunque siempre suelo tirar por lo vanilla...

Reply:
Pero en este caso no me vale porque necesito que me devuelva `null` sino existe